styleguide/containers/FormControlsContainer.jsx

```diff
@@ -1,14 +1,300 @@
-import React from "react";
-
+import React from 'react';
+import TextInput from '../../lib/components/TextInput';
+import TextArea from '../../lib/components/TextArea';
+import Dropdown from '../../lib/components/Dropdown';
+import Checkbox from '../../lib/components/Checkbox';
+import Radio from '../../lib/components/Radio';
+import FieldSet from '../../lib/components/FieldSet';
+import validators from '../../lib/utils/validators';
 import BaseContainer from "./BaseContainer";
 
-export default class FormControlsContainer extends React.Component
-{
-  render()
-  {
+export default class FormControlsContainer extends React.Component {
+  render() {
     return (
       <BaseContainer {...this.props}>
-        TODO
+        <section id="text-input">
+          <h2 className="usa-heading heading-margin-alt" id="text-input">Text input</h2>
+          <p className="usa-font-lead">Text inputs allow people to enter any combination of letters, numbers, or symbols of their choosing (unless otherwise restricted). Text input boxes can span single or multiple lines.</p>
+          <div className="preview">
+            <TextInput
+              label="Text input label"
+              id="firstInput"/>
+            
+            <TextInput
+              label="This input is required"
+              id="secondInput"
+              required />
+            
+            <TextInput
+              label="Must have an 'A' and an 'e'"
+              id="thirdInput"
+              value="Ave"
+              isValid
+              validators={[validators.hasLowercaseE, validators.hasUppercaseA]} />
```

Comment:
Let's move the definition of these sample validators into this file and leave only the "officially supported" pre-defined validators in that file.